pkgs/top-level/python-packages.nix

```diff
@@ -15649,10 +15649,10 @@ EOF
   # Should be bumped along with EFL!
   pythonefl = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
@FRidh a bot for this message would be nice :)

Reply:
https://github.com/NixOS/ofborg/issues/41